### ISSUE TITLE
[regression] FPS drop in the libretro core with Logging Verbosity ON

### ISSUE THREAD

**Issue by sergiobenrocha2** [mgba-emu/mgba#235]
Just in case it got lost in the irc.

There's a FPS drop happening only in the libretro core, Pokemon Pinball Ruby & Saphire. Go to the Pokédex and move the list up or down with no interruptions, you'll notice 34 fps

That's a regression from the refactors.

EDIT: It only happens if you have Settings Tab --> Logging --> Logging Verbosity = ON
then, you'll notice lot of this:
```
RetroArch [libretro INFO] :: GBA I/O: Read from write-only I/O register: 0DC
```

**Comment by endrift** [comment]
I don't seem to be able to reproduce this. Maybe my laptop is too powerful?

**Comment by sergiobenrocha2** [comment]
Eh... I'm using a core i7 4700HQ CPU 2.40GHz

**Comment by endrift** [comment]
Hm, my laptop is an i7-4850HQ, so that really shouldn't be a problem.

**Comment by endrift** [comment]
Can you verify if this is still present in the latest master? I fixed some things recently involving speed regressions.

**Comment by sergiobenrocha2** [comment]
Still an issue endrift

**Comment by sergiobenrocha2** [comment]
GBA bios intro has the same issue.

**Comment by endrift** [comment]
This is on Ubuntu? I haven't tried reproducing on anything other than OS X yet.

**Comment by sergiobenrocha2** [comment]
I just tested it on Ubuntu, yeah.

**Comment by sergiobenrocha2** [comment]
![pokemon pinball - ruby sapphire usa srm](https://cloud.githubusercontent.com/assets/5431285/15099656/93922efa-1531-11e6-99e6-bf3cff73c3a5.png)

**Comment by sergiobenrocha2** [comment]
![untitled](https://cloud.githubusercontent.com/assets/5431285/15099746/c2105d12-1534-11e6-8cc5-7e89bddfc0ea.png)

**Comment by sergiobenrocha2** [comment]
![untitled2](https://cloud.githubusercontent.com/assets/5431285/15099770/8eda4d94-1535-11e6-8ff4-b77fb9ccf765.png)

**Comment by endrift** [comment]
Can confirm with verbose logging enabled. However, having extra logging enabled is just generally a known bottleneck with programs, so I'm not sure it qualifies as a bug. Normal operation doesn't have logging set that high, after all.

**Comment by endrift** [comment]
Please confirm that this is fixed with a release build of 0e85739cd3fb239193704a83460b5480e0bd5862.

**Comment by sergiobenrocha2** [comment]
Fixed, I need to remember that it needs to be build with -DCMAKE_BUILD_TYPE=Release
